Sources/IBLinterKit/UseBaseClassConfig.swift

```diff
@@ -0,0 +1,36 @@
+//
+//  BaseClassConfig.swift
+//  AEXML
+//
+//  Created by masamichi on 2019/03/11.
+//
+
+import Foundation
+
+public struct UseBaseClassConfig: Codable {
+    public let elementClass: String
+    public let baseClasses: [String]
+
+    enum CodingKeys: String, CodingKey {
+        case elementClass = "element_class"
+        case baseClasses = "base_classes"
+    }
+
+    public static let `default` = UseBaseClassConfig.init()
```

Comment:
I think this `default` and `private init` is not used anywhere. Can you remove them?

Reply:
I see👍 Thanks!